yaksh/models.py

```diff
@@ -1284,6 +1284,9 @@ def get_question_answers(self):
                 }]
         return q_a
 
+    def get_latest_answer(self, question_id):
+        return self.answers.filter(question=question_id).order_by("-id").last()
```

Comment:
This looks wrong, it should be `self.answers.filter(question=question_id).order_by("id").last()`